bitrise.yml

```
@@ -0,0 +1,86 @@
+format_version: "11"
```

Let's use v24 to make sure we support regex correctly (not documented, but trust me)

```suggestion
format_version: "24"
```